pylint/testutils/lint_module_test.py

```diff
@@ -163,10 +163,10 @@ def _get_expected(self):
             ]
         return expected_msgs, expected_output_lines
 
-    def _get_actual(self):
+    def _get_actual(self) -> Tuple[Counter, List[OutputLine]]:
         messages = self._linter.reporter.messages
         messages.sort(key=lambda m: (m.line, m.symbol, m.msg))
-        received_msgs = Counter()
+        received_msgs: Counter = Counter()
```

Comment:
The annotation should not be necessary

Reply:
`Mypy` complains when it is missing. Want me to add an ignore?